.github/workflows/server-ci-artifacts.yml

```diff
@@ -0,0 +1,131 @@
+name: Server CI Artifacts
+
+on:
+  workflow_run:
+    workflows:
+      - "Server CI PR"
+    types:
+      - completed
+
+jobs:
+  ## We only need the condition on the first job
+  ## This will run only when a pull request is created with server changes
+  update-initial-status:
+    if: github.repository_owner == 'mattermost' && github.event.workflow_run.event == 'pull_request' && github.event.workflow_run.conclusion == 'success'
```

Comment:
If I fork and edit this file to remove the `repository_owner` check, does anything bad happen when I push the PR?

Reply:
@lieut-data  

Why deny inheritance of Secrets when running in forks. 
Moreover the GITHUB_TOKEN secret has read-only permissions in pull requests from forked repositories.